content/backend/graphql-scala/6-interfaces.md

```diff
@@ -279,26 +279,26 @@ query {
 }  
 ```
 
-### Finding a common parts
+### Finding common parts
 
-As you can see parts that are very similar. Like `HasId` for all three types:
+As you can see some code is very similar. Like `HasId` for all three types:
 
 ```scala
 implicit val linkHasId = HasId[Link, Int](_.id)
 implicit val userHasId = HasId[User, Int](_.id)
 implicit val voteHasId = HasId[Vote, Int](_.id)
 ```
 
-What if you want to add more entities? You will duplicate code even more.
+What if you want to add more entities? You will have to duplicate this code.
 
 The solution for this is an interface. We can provide an interface that will be extended by any of the entities. This way, for example, you will need just one HasId
 
-<Instrunction>
+<Instruction>
 
-Create trait `Identifable`:
+Create trait `Identifiable`:
 
 ```scala
-trait Indentifable {
+trait Identifiable {
```

Comment:
I still can't figure out how this trait connects to `IdentifiableType` in the subsequent section.

Reply:
https://github.com/howtographql/howtographql/pull/669
i opened new PR which includes the interface definition.